cmd/csi-node-driver-registrar/node_register.go

```diff
@@ -126,6 +136,43 @@ func httpServer(socketPath string, httpEndpoint string) {
 	klog.Fatal(http.ListenAndServe(httpEndpoint, mux))
 }
 
+func checkLiveRegistrationSocket(socketFile, csiDriverName string) error {
+	klog.V(1).Infof("Attempting to open a gRPC connection with: %q", socketFile)
+	cmm := metrics.NewCSIMetricsManagerForSidecar("")
+	csiConn, err := connection.Connect(socketFile, cmm)
```

Comment:
it's not CSI connection, it's registration socket

Reply:
renamed the socket file.